### ISSUE TITLE
Problem rendering some "embarcadero samples" icons

### ISSUE THREAD

**Issue by carloBarazzetta** [EtheaDev/SVGIconImageList#51]
In past, all the icons present into folder "D:\ETHEA\GitHub_EtheaDev\SVGIconImageList\Demo\NewSydneyVectors" where rendered correctly.
Now RAD_SERVER_Product icon.svg and C++Builder project icon.svg is not rendered correctly.
I remember that I've fixed a problem of initialitazion of svg color from black to inherit... but I'm not sure...
@pyscripter , can you check this problem?

**Comment by carloBarazzetta** [comment]
for example, I remember that "C++Builder project icon.svg" problem was the definition of st2 style, as display none and stroke black.
the "circle" with st2 was rendered black, but it shouldn't be rendered because the style is display none.
Now I'm searching the bug reintroduced probably with some merge...

**Comment by pyscripter** [comment]
Here is how it is shown here.  It shows the same in the browser.

![image](https://user-images.githubusercontent.com/1311616/90868722-e3d65080-e39f-11ea-8faf-7fdad851ade6.png)

**Comment by pyscripter** [comment]
And here are all the product icons:

![image](https://user-images.githubusercontent.com/1311616/90869027-55160380-e3a0-11ea-977c-8180dc338c0b.png)

**Comment by carloBarazzetta** [comment]
Yes, take a look at 85.C++ Builder project icon, notice the "black" circle rendered.

> Here is how it is shown here. It shows the same in the browser.
> 
> ![image](https://user-images.githubusercontent.com/1311616/90868722-e3d65080-e39f-11ea-8faf-7fdad851ade6.png)

Is not the correct icon, see below...

**Comment by carloBarazzetta** [comment]
> And here are all the product icons:
> 
> ![image](https://user-images.githubusercontent.com/1311616/90869027-55160380-e3a0-11ea-977c-8180dc338c0b.png)

Notice the problem into icon 285 and 290.
For 285 I've found the problem: an SVG with Display false is rendered, so with this fix the "black circle" is not rendered:
if (FPath = nil) or (FDisplay = tbFalse) {or (FPath.GetLastStatus <> OK)} then (line 962 of SVG.pas)

**Comment by carloBarazzetta** [comment]
For RAD_SERVER_Product icon.svg the problem is more complex:
for example the section:
<g id="Camada_3" class="st3">
	<g>
		<path class="st4" d="M1069.58,1964.76c0,0-184.86,1236.13,1199.79,1512.77c319.84,63.9,1472.02,279.11,1472.02,279.11"/>
	</g>
</g>
produce a blue line, because class st4 define the stroke as blue.
But the class st3 used into "id" node define display:none;
Any idea how to fix it?
